app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,102 +1,86 @@
+// @ts-nocheck
```

Comment:
Why have you added this?

Reply:
Was getting this problem in VS Code:

<img width="625" alt="image" src="https://user-images.githubusercontent.com/35058362/175300082-4213ab08-154a-4bc1-b5ed-220c17b3470b.png">

Although I can see the below path mentioned in `src>jsconfig.json` file

```
    "paths": {
      "neetoui": ["@bigbinary/neetoui"]
    },
```

Reply:
@sachinjoshi12 Ignore the errors with import statements or anything related to neetoUI. That error will only be shown in VSCode or any other text editor that uses LSP. If you run eslint from cli, this error won't show up. This issue will be fixed in neetoUI itself some time in the future.

>//@ts-nocheck

Adding this line would disable all error linting within that file, where there might be other relevant and helpful error lints.